docs/contributing_dev.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Contributing to DEV
+---
+
+# Contributing to dev.to
+
+- [Contributing](#contributing)
+  - [Where to contribute](#where-to-contribute)
+  - [How to contribute](#how-to-contribute)
+  - [Contribution guideline](#contribution-guidelines)
+    - [Create an issue](#create-an-issue)
+    - [Please include tests](#please-include-tests)
+    - [Code quality](#code-quality)
+    - [Create a pull request](#create-a-pull-request)
+    - [Pull requests reviews and "force pushing"](#pull-requests-reviews-and-force-pushing)
+  - [The bottom line](#the-bottom-line)
+
+## Contributing
+
+We expect contributors to abide by our underlying
+[code of conduct](https://dev.to/code-of-conduct). All conversations and
+discussions on GitHub (issues, pull requests) and across
+[dev.to](https://dev.to) must be respectful and harassment-free.
+
+Remember that communication is the lifeblood of any Open Source project. We are
+all working on this together, and we are all benefiting from this software. It's
+very easy to misunderstand one another over asynchronous, text-based
+conversations: When in doubt, assume everyone you're interacting within this
+project has the best intentions.
+
+If you feel another member of the community has violated our Code of Conduct,
+you may anonymously contact the team with our
+[abuse report form](https://dev.to/report-abuse).
+
+### Where to contribute
+
+All [issues](https://github.com/thepracticaldev/dev.to/issues) labeled
+[ready for dev](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22ready+for+dev%22)
+and
+[type: bug](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22)
+all are up for grabs.
+
+- [good first issue](https://github.com/thepracticaldev/dev.to/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+)
+  are issues meant for newer developers.
+- [difficulty: easy](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+easy%22)
+  issues are usually confined to isolated areas of existing code.
+- [difficulty: medium](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+medium%22)
+  issues are usually can sometimes entail new features and might affect a
```

Comment:
"issues are usually can sometimes " There seems to be an extra word here.

Reply:
Thanks! Copy/paste got me :stuck_out_tongue: